files/hola_player/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "npm",
+  "name": "@hola.org%2fhola_player",
```

Comment:
I tested it via libgrabber and it was only way to use it. How I can pass npm-name like this:
'@hola.org/hola_player'?